src/govuk/components/summary-list/_index.scss

```diff
@@ -166,4 +166,72 @@
       }
     }
   }
+
+  // Additional block for the summary card
+  //
+  // We ship this with the summary list instead of making it it's own component
+  // because it's only interacted with via the summary list and it avoids awkwardness
+  // with trying to document a "private" component that's controlled via the
+  // component it's wrapping
```

Comment:
Is this important enough to include as a comment here? It's useful context, but not going to be relevant for most people. Wondering if it'd be better off living in the the commit message.

Reply:
I agree with this

Reply:
I'll move to the commit message 👍🏻